dallo/src/helpers.rs

```diff
@@ -4,31 +4,27 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use crate::SCRATCH_BUF_BYTES;
-
+use crate::{StandardDeserialize, SCRATCH_BUF_BYTES};
 use rkyv::ser::serializers::{
     BufferScratch, BufferSerializer, CompositeSerializer,
 };
 use rkyv::ser::Serializer;
-use rkyv::{archived_root, Archive, Deserialize, Serialize};
+use rkyv::{check_archived_root, Archive, Deserialize, Serialize};
```

Comment:
Unless there is a specific reason, it usually in Rust it's more idiomatic having `checked` / `unchecked` as suffix instead of prefix (see the standard library for example).

Reply:
This is the name in `rkyv`, not something we control.